chanzuckerberg/idseq-web
# Add tooltip on save button click

# Description

![image](https://user-images.githubusercontent.com/28797/55028384-c80d2600-4fc4-11e9-8cb5-cda8f948c9dd.png)

The UEX was broken because there was no feedback on save. This adds a tooltip same as the "share" button. 

This also fixes a bug that was introduced when switching to `BasicPopup`, which ignored the `on=` attribute of the tooltip. 

I also changed the `pushState` updating of the URL to `replaceState`, following @jshoe 's fix on the report page. We'd need to do some work to get the back button working within visualizations. 

NOTE: the experience could still be better because it shows optimistically, unconditional on success, and the color stays the same while focussed, keeping with the current site wide color scheme for `Button`. 

# Test

1. Click save on heatmap, phylo tree and taxon tree
2. See tooltip message
3. repeat for share click
